src/Microsoft.Health.Dicom.SqlServer/Microsoft.Health.Dicom.SqlServer.csproj

```diff
@@ -21,18 +25,40 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.Health.Dicom.Core\Microsoft.Health.Dicom.Core.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Features\CustomTag\" />
+    <Folder Include="Features\Schema\Model\" />
+  </ItemGroup>
```

Comment:
What are these `<Folder/>` elements for?

Reply:
It's added by Visual Studio, let me see if can get rid of.